### PR TITLE
Add aliases in nomination

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/dotnet/project-system/issues/6011. 

Feature design at: https://github.com/NuGet/Home/blob/dev/designs/PackageReference-Extern-Alias.md

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/project-system/pull/6130)

### CODE REVIEW

**Comment by davidwengier** [comment]
As per the test failure, you need to build to regen the xlf files.

Thank you for keeping the properties in alphabetical order <3

**Comment by nkolev92** [comment]
Thanks @davidwengier 

Fixed the xlfs.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml [hunk 1]
```diff
@@ -13,6 +13,10 @@
                 SourceOfDefaultValue="AfterContext"
                 SourceType="TargetResults" />
   </Rule.DataSource>
+  
+  <StringProperty Name="Aliases"
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml:17]: This also needs to be added to ResolvedPackageReference.xml

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml [hunk 2]
```diff
@@ -13,6 +13,10 @@
...
+                  Description="Aliases for the assembly references coming from this PackageReference."
+                  DisplayName="Aliases" />
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml:19]: Remove DisplayName, it doesn't need to be present if names matches.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml [hunk 3]
```diff
@@ -13,6 +13,10 @@
                 SourceOfDefaultValue="AfterContext"
                 SourceType="TargetResults" />
   </Rule.DataSource>
+  
+  <StringProperty Name="Aliases"
+                  Description="Aliases for the assembly references coming from this PackageReference."
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml:18]: "coming from this package.".

**Comment by nkolev92** [comment]
@davkean I'll create a follow up PR to address the feedback and the additions ResolvedPackageReference.xaml.

**Comment by davkean** [comment]
@nkolev92 I'm fixing this, don't worry - I missed that this was merged.

**Comment by nkolev92** [comment]
Sounds good., thanks @davkean

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml (hunk 2 above)
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml:19]: It does if you want it to be localised. We have a test that will fail if removed, IIRC.

**Pull request merged**